lib/foods.js

```diff
@@ -24,5 +24,19 @@ Food.prototype.delete = function () {
   localStorage.setItem('foods', foodsJSON);
 };
 
+Food.prototype.edit = function (oldName, oldCalories) {
+  var foodsJSON = localStorage.getItem('foods');
+  var foods = JSON.parse(foodsJSON);
+  var newindex = ''
```

Comment:
@MDes41 slightly confused about this newIndex variable name... I guess i'm knit picking but its just finding the same index to make sure thats the right element in the array to edit... ⛏